test/valgrind/valgrindTestSuite.sh

```diff
@@ -456,6 +472,25 @@ function failedTest()
 }
 
 
+
+# -----------------------------------------------------------------------------
+#
+# memErrorFound
+#
+function memErrorFound()
+{
+  _valgrindFile=$1
+  _file=$2
+  _memErrors=$3
+
+  echo "FAILED (mem errors: $_memErrors). Check $_valgrindFile for clues"
```

Comment:
What would happen if at the same time a given test fails due to leaks *and* invalid writes/reads? Which sort of output would we see?